zookeeper.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: zookeeper
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - site_vars.yml
+  roles:
+    - role: pulibrary.openjdk8
+    - role: pulibrary.samba
```

Comment:
None of the variables used to configure samba (samba_shares_root, samba_shares, samba_users, etc...) are setup for these roles. Is this what we want? What are we using samba for here?